distributed/utils_test.py

```diff
@@ -891,8 +891,9 @@ async def start_cluster(
     scheduler_kwargs: dict[str, Any] = {},
     worker_kwargs: dict[str, Any] = {},
 ) -> tuple[Scheduler, list[ServerNode]]:
+    if loop is not None:
+        warnings.warn("the loop kwarg to start_cluster is deprecated")
```

Comment:
Since this is an internal test function, it seems better to me to just remove the argument now without a deprecation warning.